src/presenters/title-presenter.php

```diff
@@ -12,46 +12,42 @@
 /**
  * Class Title_Presenter
  */
-class Title_Presenter extends Abstract_Indexable_Presenter {
+class Title_Presenter extends Abstract_Indexable_Tag_Presenter {
 
 	/**
-	 * Returns the document title.
+	 * The tag format including placeholders.
 	 *
-	 * @param bool $output_title_tag Optional. Whether or not to output the title with the title tag (`<title>`) included. Defaults to true.
-	 *
-	 * @return string The document title tag.
+	 * @var string
 	 */
-	public function present( $output_title_tag = true ) {
-		$title = $this->replace_vars( $this->presentation->title );
-		$title = $this->filter( $title );
-		$title = $this->helpers->string->strip_all_tags( \stripslashes( $title ) );
-		$title = \trim( $title );
-
-		if ( \is_string( $title ) && $title !== '' ) {
-			if ( $output_title_tag ) {
-				return '<title>' . \esc_html( $title ) . '</title>';
-			}
-			return \esc_html( $title );
-		}
-
-		return '';
-	}
+	protected $tag_format = '<title>%s</title>';
 
 	/**
-	 * Run the document title through the `wpseo_title` filter.
+	 * Gets the raw value of a presentation.
 	 *
-	 * @param string $title The document title to filter.
-	 *
-	 * @return string $title The filtered document title.
+	 * @return string The raw value.
 	 */
-	private function filter( $title ) {
+	public function get() {
+		$title = $this->replace_vars( $this->presentation->title );
 		/**
 		 * Filter: 'wpseo_title' - Allow changing the Yoast SEO generated title.
 		 *
 		 * @api string $title The title.
 		 *
 		 * @param Indexable_Presentation $presentation The presentation of an indexable.
 		 */
-		return \apply_filters( 'wpseo_title', $title, $this->presentation );
+		$title = \apply_filters( 'wpseo_title', $title, $this->presentation );
+		$title = $this->helpers->string->strip_all_tags( \stripslashes( $title ) );
+		return \trim( $title );
+	}
+
+	/**
+	 * Escaped the output.
+	 *
+	 * @param string $value The value.
+	 *
+	 * @return string The escaped value.
+	 */
+	protected function escape( $value ) {
```

Comment:
If we add the functionality below to the abstract based on the `$escaping` class attribute, this isn't needed, right?